website/static/js/registrationUtils.js

```diff
@@ -48,16 +48,33 @@ function Comment(data) {
     self.lastModified = new Date(data.lastModified)|| new Date();
     self.value = ko.observable(data.value || '');
 
+	self.isDeleted = ko.observable(false);
+	self.isDeleted.subscribe(function(isDeleted) {
+		if (isDeleted) {
+			self.value('this comment was deleted');
+
+		}
+	});
+
+	self.seenBy = ko.observableArray([]);
+
     /**
-     * Returns 'You' if the current user is the commenter, else the commenter's name
+     * Saves the author as the actual user, not you 
```

Comment:
It's true that the computed doesn't save anything but comment.author is what is saved in the schema. That meant that it always showed up as 'You'. Save is not the best choice of words.